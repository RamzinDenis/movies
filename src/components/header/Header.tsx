import React from "react";
import Navigation from "../navigation";
import logo from "../../asserts/images/reactMovie_logo.png";
import styles from "./header.module.sass";

const Header: React.FC = () => {
	return (
		<header className={styles.header}>
			<div className={styles.container}>
				<img src={logo} className={styles.logo} alt="logo" />
				<Navigation currentUser={false} />
			</div>
		</header>
	);
};

export default Header;
