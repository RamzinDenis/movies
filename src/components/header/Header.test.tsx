import React from "react";
import Header from "./";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("<Header />", () => {
	it("should render Header correctly", () => {
		const component = shallow(<Header />);
		expect(toJson(component)).toMatchSnapshot();
	});
});
