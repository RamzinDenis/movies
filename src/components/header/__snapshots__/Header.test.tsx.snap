// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render Header correctly 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <img
      alt="logo"
      className="logo"
      src="reactMovie_logo.png"
    />
    <withRouter(Navigation)
      currentUser={false}
    />
  </div>
</header>
`;
